apps/rewards/app/components/App/App.js

```diff
@@ -1,27 +1,51 @@
-import { AragonApp, observe, SidePanel } from '@aragon/ui'
+import { AragonApp, observe, SidePanel, TabBar } from '@aragon/ui'
 import PropTypes from 'prop-types'
 import React from 'react'
 import styled from 'styled-components'
 import { Title } from '../Shared'
+import { Empty } from '../Card'
 
 const ASSETS_URL = 'aragon-ui-assets/'
 
 class App extends React.Component {
   static propTypes = {
     app: PropTypes.object.isRequired,
-    accounts: PropTypes.arrayOf(PropTypes.object),
+    rewards: PropTypes.arrayOf(PropTypes.object),
+  }
+
+  state = {
+    selected: 0,
+    rewardsEmpty: this.props.rewards === undefined || this.props.rewards === []
   }
 
   closePanel = () => {
     this.setState({ panel: { visible: false } })
   }
 
+  selectTab = idx => {
+    this.setState({ selected: idx })
+  }
+
+  onNewReward = () => {
+    console.log('Create New Reward')
+  }
+
   render() {
 
     return (
       <StyledAragonApp>
         <Title text="Rewards" />
+        <TabBar
+          items={[ 'Overview', 'My Rewards' ]}
+          selected={this.state.selected}
```

Comment:
Please create those tabs? In separate files, like components/Content in Projects.

Reply:
I've created them and they're hooked up to the tabs as well.